pkg/occlient/occlient.go

```diff
@@ -1328,31 +1338,121 @@ func (c *Client) GetClusterServiceClasses() ([]scv1beta1.ClusterServiceClass, er
 }
 
 // CreateServiceInstance creates service instance from service catalog
-func (c *Client) CreateServiceInstance(componentName string, componentType string, labels map[string]string) error {
-	// Creating Service Instance
-	_, err := c.serviceCatalogClient.ServiceInstances(c.namespace).Create(
+func (c *Client) CreateServiceInstance(serviceName string, serviceType string, servicePlan string, parameters map[string]string, labels map[string]string) error {
```

Comment:
I don't believe that in this case we should use `componentName` and `componentType`. IMHO it will create confusion, since this function deals specifically with the ServiceCatalog's service